### ISSUE TITLE
Does the bn_layer contain scale and shift?

### ISSUE THREAD

**Issue by githubfragments** [alexgkendall/caffe-segnet#108]
The official caffe version's batch_norm_layer has not contain scale and shift, and need to add additional scale layer to complete batch normalization. 
Does this bn_layer contain scale and shift?

**Comment by TimoSaemann** [comment]
Yes it does.

**Comment by yifaliu** [comment]
@TimoSaemann does it mean BN layer in segNet is the same as 'batch normalization + scale layer' in official caffe version?

**Comment by TimoSaemann** [comment]
https://github.com/alexgkendall/caffe-segnet/issues/109

**Comment by githubfragments** [comment]
Good, thank you @TimoSaemann
